apps/src/dance/ai/DanceAiModal.tsx

```diff
@@ -396,6 +405,7 @@ const DanceAiModal: React.FunctionComponent<DanceAiProps> = ({onClose}) => {
               onTypingDone={() => {
                 setTypingDone(true);
                 currentAiModalField?.setValue(responseJson);
+                setShowPreview(true);
```

Comment:
This is probably more of a question for @samantha-code.  Now that we show the preview in the modal, should we wait to show the ai-generated background in the visualization column until modal is closed?
For example, if there are already background/foreground effects in the `setup` block, and then we click and drag a new ai block, I wonder if it's a little confusing to show the new generated background in both the modal and vis column before the user actually selects 'use' or 'convert'.

Reply:
Current:


https://github.com/code-dot-org/code-dot-org/assets/107423305/d862e378-4c32-4717-97f8-92a1569e87a1



If we don't show till after modal is closed:

https://github.com/code-dot-org/code-dot-org/assets/107423305/a44b5319-be16-49fb-885b-fb0eb8757392

Reply:
For after screencast, moved `currentAiModalField?.setValue(responseJson);` to `handleDoneClick`.